packages/patternfly-4/content/design-guidelines/usage-and-behavior/filters/design.md

```diff
@@ -0,0 +1,217 @@
+# Filters
```

Comment:
Can you add this at the top of the file
```
---
path: "/design-guidelines/usage-and-behavior/filters"
---
```
Then in the usage-and-behavior folder you will also see a json file, need to add the new navigation item in there
```
{
        "text": "Filters",
        "path": "/design-guidelines/usage-and-behavior/filters"
      }, 
```

Reply:
All set 👍